lib/ap/internal/fcall.h

```diff
@@ -30,13 +30,13 @@ enum
 struct	Fcall
 {
 	char	type;
-	short	fid;
-	unsigned short	tag;
+	int16_t	fid;
```

Comment:
oh and they're 32 bits now, so I don't know why its 16 bits. here. 16 bits is old9p.